Fix/env warning

DAsfLeanIX opened the pull request.

perbernhardt commented on package.json:
```diff
@@ -4,7 +4,8 @@
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npx @vercel/ncc build index.js"
```
npx! Nice :-)

perbernhardt commented on update-dist.sh:
```diff
@@ -0,0 +1,3 @@
+npm i -g @vercel/ncc
```
I think we can remove update-dist.sh completely as you use the package/scripts option to define the call.

DAsfLeanIX replied:
It's removed now

perbernhardt reviewed (changes requested):
... and please remove the "node_modules" from git completely and also add it to the .gitignore. Thanks!

DAsfLeanIX commented:
> ... and please remove the "node_modules" from git completely and also add it to the .gitignore. Thanks!

Removed the node_modules and added the folder to gitignore!
